server/tests/test_race_conditions.py

```
@@ -0,0 +1,277 @@
+        await webhook_service.process_call_report(webhook_data, test_db)
+        
+        # Re-fetch both call and service_record
+        call_result = await test_db.execute(
+            select(Call).where(Call.id == call_id)
+        )
+        call = call_result.scalar_one()
+        
+        sr_result = await test_db.execute(
+            select(ServiceRecord).where(ServiceRecord.id == sr_id)
+        )
+        service_record = sr_result.scalar_one()
```

Ruff (0.14.7)

Local variable `service_record` is assigned to but never used

Remove assignment to unused variable `service_record`

(F841)